apps/onboarding/src/app/result/page.tsx

```diff
@@ -82,34 +84,53 @@ const Volumn = styled.p`
 `;
 
 const Result = () => {
+  const pathname = usePathname();
   const searchParams = useSearchParams();
+  const drinkType = searchParams.get('drinkType');
   const glasses = Number(searchParams.get('glasses'));
   const { name, svg, description, mainColor } = getLevelDetails(glasses);
+  const { isLoading, data } = useQuery(['result'], () => {
+    return axios
+      .get(
+        `https://sulsul.app/api/v1/drinkingLimit/share?drinkType=${drinkType}&glass=${glasses}`
+      )
+      .then((response) => response.data);
+  });
+
+  const shareKakao = () => {
+    window.Kakao.Share.sendScrap({
+      requestUrl: `https://sulsul.app${pathname}?glasses=${glasses}`,
+    });
+  };
+
+  if (isLoading) return <div>loading...</div>;
 
   return (
     <Page>
       <Heading2>당신은...</Heading2>
       <ResultCard name={name} svg={svg} description={description} mainColor={mainColor} />
       <Heading3>다른 술은 얼마나 마실 수 있을까?</Heading3>
       <DrinkLists>
-        {Object.values(DrinkResDrinkTypeEnum).map((alcohol) => {
-          const { name, svg, volumn } = AlcoholDetails[alcohol];
+        {data.otherDrinks.map((drink: DrinkRes) => {
```

Comment:
DrinkRes은 백엔드에서 사용하는 객체의 이름이기 때문에 drinkRes로 그대로 써주면 더 좋을 거 같아요

```suggestion
        {data.otherDrinks.map((drinkRes: DrinkRes) => {
```